### DISCUSSION TITLE
Speed Editor & Cubase menu

### DISCUSSION THREAD

**Discussion by Folleville** [CommandPost/CommandPost#3228] [Technical Support]
Hi,

Thank you for providing this very useful tool. I am trying to use the Resolve Speed Editor together with Cubase using CommandPost, but I am a little lost about how I can map keys to menu actions in Cubase. It seems CommandPost is trying to fetch the menus from Cubase and gets stuck. Could it be because of workspaces and fullscreen apps?

I am probably missing something somewhere?

Regards

**Comment by latenitefilms** [comment]
Make sure you enable the two **Search Console** options in the General Preferences:

<img width="1000" alt="image" src="https://github.com/CommandPost/CommandPost/assets/22286696/7bc28249-65ec-4bbb-a99c-34cd9eab3e28">

**Reply by Folleville** [reply]
Hi,

Sorry for taking so long but I needed the time to perform more tests and ensure my configuration is as you mentioned. I can confirm the following : 
- the two boxes are ticked
- it fetches menus properly for other apps (say Outlook)
- it hangs on "loading" when trying the Cubase app. It does detect it is running as I have a different message when it is not (the menus will be loaded when the application is running)

Could it be because the menus are too long?

Regards!

**Reply by latenitefilms** [reply]
Menu size shouldn't really matter, and I doubt Cubase has more menu items than something like DaVinci Resolve.

Are there any errors in the Debug Console after scanning?

Can you share an entire screen screenshot of Cubase and its biggest menu?

Have you tried quitting CommandPost and reopening it whilst Cubase is already running and has a project active?

**Reply by Folleville** [reply]
Thank you for your timely reply.

Here are the pictures attached. I did try quitting CommandPost and restarting it while Cubase was running, to no avail.

I do not see any errors after scanning, but I don't think scanning ever completes since it hangs on "Loading"

<img width="506" alt="Capture d’écran 2023-09-29 à 22 21 40" src="https://github.com/CommandPost/CommandPost/assets/137885555/de76f93d-04d5-45d9-9c99-21d4992248ca">
<img width="1680" alt="Capture d’écran 2023-09-29 à 22 19 16" src="https://github.com/CommandPost/CommandPost/assets/137885555/4441fbb4-14b6-4397-aac9-2e131609b42f">
<img width="1680" alt="Capture d’écran 2023-09-29 à 22 20 42" src="https://github.com/CommandPost/CommandPost/assets/137885555/d0372984-97b4-46b5-9a04-39b70cdc2e0e">

**Reply by latenitefilms** [reply]
The "Loading..." is in the search field. It looks like you might have accidentally pasted it there? Try removing it, to clear the search filter. Currently it's looking for an item called "Loading...".

**Reply by Folleville** [reply]
Now I feel stupid, surely I must have pasted it there on my very early discovery of the tool, my bad. Thanks for helping! Now I can access the menus!

However, it is weird that it would be persistent, even after resetting the Cubase application in the Resolve panel, and after upgrading CommandPost, it was still there when opening that dialogue.

**Reply by latenitefilms** [reply]
If you right click on the Search Console, I believe there's an option to remember the last search query, which you must have enabled.
